src/server/mod.rs

```diff
@@ -120,7 +120,7 @@ enum PollMain {
 // ===== impl Server =====
 
 impl<S, E, B> Server<S, E, B>
-where S: NewService<Request = Request<RecvBody>, Response = Response<B>>,
+where S: NewService<Request<RecvBody>, Response = Response<B>>,
```

Comment:
I know that this is semantically the change from https://github.com/tower-rs/tower/pull/109, but it does make me wonder why we treat `Request` as special, but not `Response`. I guess in some sense, `Request` is the choice of the user, whereas the `Response` is a property of the `Service`, but it still just _feels_ weird.

Reply:
Yeah, I agree that this feels strange. However, I _think_ that semantically it's the right thing --- you'd expect a `Service` to be able to accept a set of multiple (probably bounded) request types, but respond with a fixed response type that's specified at the `Service` implementation. It _looks_ weird, but semantically feels more natural IMO.